projects/angular/src/data/stack-view/_stack-view.clarity.scss

```
@@ -86,20 +88,13 @@
     }
 
     .stack-block-label {
-      padding: tokens.$cds-global-space-5 tokens.$cds-global-space-6 tokens.$cds-global-space-5
-        tokens.$cds-global-space-11;
+      padding: density.$clr-base-vertical-offset-m density.$clr-base-horizontal-offset-l
+        density.$clr-base-vertical-offset-m
+        calc(density.$clr-base-icon-size-s + density.$clr-base-gap-s + density.$clr-base-horizontal-offset-l);
```

This token should be alright. 
```suggestion
        density.$clr-base-layout-space-xl;
```